sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Listeners/ServiceBusScaleMonitorTests.cs

```diff
@@ -137,6 +154,198 @@ public void GetMetrics_ReturnsExpectedResult()
             Assert.AreNotEqual(default(DateTime), metrics.Timestamp);
         }
 
+        [Test]
+        public async Task GetMetrics_IgnoresScheduledMessages()
+        {
+            var scheduledMessage = ServiceBusModelFactory.ServiceBusReceivedMessage(serviceBusMessageState: ServiceBusMessageState.Scheduled);
+
+            var anotherScheduledMessage = ServiceBusModelFactory.ServiceBusReceivedMessage(serviceBusMessageState: ServiceBusMessageState.Scheduled);
+
+            _mockMessageReceiver.Setup(x => x.PeekMessageAsync(It.IsAny<long>(), It.IsAny<CancellationToken>()))
```

Comment:
Is it possible to update the mocks in these tests to specify 0 for the starting number, which is what we're using? If it's possible to specify the sequence number of the first message, you might then also update the batch peek mock to include both that starting sequence as well as the batch count?

Reply:
My idea was to have a dedicated test that verifies the sequence number and keep the other one's free from it. Following that philosophy, though, I'm missing one for the batch size. Will add

Reply:
Thanks for the kudos :D